src/cascadia/TerminalSettingsModel/GlobalAppSettings.h

```diff
@@ -102,21 +97,19 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         INHERITABLE_SETTING(Model::GlobalAppSettings, bool, DetectURLs, true);
         INHERITABLE_SETTING(Model::GlobalAppSettings, bool, MinimizeToTray, false);
         INHERITABLE_SETTING(Model::GlobalAppSettings, bool, AlwaysShowTrayIcon, false);
+        INHERITABLE_SETTING(Model::GlobalAppSettings, winrt::Windows::Foundation::Collections::IVector<winrt::hstring>, DisabledProfileSources, nullptr);
+        INHERITABLE_SETTING(Model::GlobalAppSettings, hstring, UnparsedDefaultProfile, L"");
 
     private:
-        guid _defaultProfile;
-        std::optional<hstring> _UnparsedDefaultProfile{ std::nullopt };
-        bool _validDefaultProfile;
-
-        com_ptr<implementation::ActionMap> _actionMap;
+#ifdef NDEBUG

```

Comment:
Curious -- why switch from `!_DEBUG` to `NDEBUG`? Is one more "official"?

Reply:
`_DEBUG` is a MSVC thing. C++ actually only specifies `NDEBUG` for "no debug assertions" (which has mutated to "release build" nowadays). So yeah I simply switched to the official macro there.